index.html

```diff
@@ -110,6 +110,28 @@ <h3 class="works_headline">My Recent Works</h3>
           class="works_headline_indicator"
         />
       </div>
+      <!-- CARD WORK PROJECT DETAILS -->
+      <div id="work_project_details" class="work_project_details">
+        
+        <div class="work_project_details_card">
+          <span class="work_detail_close_icon">&times; </span>
+          <img class="work_project_details_img" src="./media/images/projects/youtube.PNG" alt="project screenshot"/>
+            <div class="work_project_details_title_btns"><h5 class="work_project_details_title"></h5>
+              <div class="work_project_details_btn_container btns_title">
```

Comment:
- [ ] Please add the stack of technologies used for the project below the project tile:

Yours  | Figma
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/151188228-4a93642a-cfff-4bfd-a2f3-3f5c7278ed03.png)| ![image](https://user-images.githubusercontent.com/77038610/151188187-9ee795fc-e6fd-4847-801c-863e02190ff6.png)

As you can see, yours is currently empty.